src/app/shared/navigation-focus/navigation-focus.ts

```diff
@@ -13,7 +13,8 @@ export class NavigationFocus implements OnInit {
   ngOnInit() {
     clearTimeout(lastTimeoutId);
     // 100ms timeout is used to allow the page to settle before moving focus for screen readers.
-    lastTimeoutId = setTimeout(() => this.el.nativeElement.focus({preventScroll: true}), 100);
+    lastTimeoutId = window.setTimeout(() => this.el.nativeElement.focus({preventScroll: true}),
```

Comment:
For some reason the build always failed here because it thought this was referring to NodeJS's function which returns a `NodeJS.Timeout` or `NodeJS.Timer`. Adding `window` was the widely accepted solution across StackOverflow and issue trackers.

Reply:
cc @devversion the nodejs typings shouldn't be making their way into the web compilation unit, but this is something we can deal with later